docs/sources/reference/run.md

```diff
@@ -451,6 +451,19 @@ familiar with using LXC directly.
 > you can use `--lxc-conf` to set a container's IP address, but this will not be
 > reflected in the `/etc/hosts` file.
 
+## Logging drivers (--log-driver)
+
+It is possible to specify a different logging driver for the container than the
```

Comment:
Eliminate passive voice: You can specify a different logging driver for the container...